app/src/main/java/com/github/multimatum_team/multimatum/adaptater/DeadlineAdapter.kt

```diff
@@ -138,7 +138,6 @@ class DeadlineAdapter(
             }
             clockService.now() > deadline.dateTime -> {
                 detail = context.getString(R.string.isAlreadyDue)
-                detailTextView.setTextColor(Color.BLACK)
```

Comment:
As we set the color of this text to green and red in another part of the code and the addition of the done button last week, we can have a problem.
Suppose that we have a deadline done who is already due. The color of this text should be green and show "Done". However, if the user press the done button, it will change the state of deadline to TODO and this text too.
As the deadline is already due, the text will noe be "Is already due" **but the text will still be in green** as we don't change the color. This setTextColor was to resolve this (but I didn't thought about the darkmode).
To make this work, I think you can put this instead
`detailTextView.setTextColor(getResources().getColor(R.color.deadline_details_title))`

Reply:
Yes I will try that. The function that you suggest is deprecated but I will try to do it with the actual equivalent.